forest/src/cli/chain_cmd.rs

```diff
@@ -17,6 +20,22 @@ pub enum ChainCommands {
         cid: String,
     },
 
+    /// Export a snapshot of the chain to <output_path>
+    #[structopt(about = "Export chain snapshot to file")]
+    Export {
+        #[structopt(short, help = "Tipset to start the export from, default is @HEAD")]
```

Comment:
If instead of `help = "...` we could put it as a doc comment. This way it'd be visible in documentation (or IDE prompts).

Reply:
Fixed.